server/services/functionalMailboxService.ts

```diff
@@ -55,7 +55,7 @@ export class FunctionalMailboxService {
   ) {}
 
   getAllProbationAreas = async () => {
-    const [{ content: probationAreas = [] } = {}, probationAreaCodes = []] = await Promise.all([
+    const [{ content: probationAreas } = { content: [] }, probationAreaCodes = []] = await Promise.all([
```

Comment:
meh.. would have been tempted to push quirks of the dependent service down in to client - so the client just always returns a list.

I'm going to have a quick look at fixing in community-api but doesn't need to hold up this PR.

Reply:
actually this looks a little weird - the client for `getAllTeamsForLdu` does handle the quirks in the service layer but `getAllProbationAreas` doesn't and has the defaulting at this level?